### ISSUE TITLE
Support for specifying 2 values for border radius

### ISSUE THREAD

**Issue by chrisdrackett** [Compass/compass#165]
currently it seems that border radius only takes 1 value. It should be possible to specify 2 values (horizontal, and vertical)

**Comment by chrisdrackett** [comment]
i.e.

```
-moz-border-radius-topleft:     1px 5px;
-webkit-border-top-left-radius: 1px 5px;
```

**Comment by chriseppstein** [comment]
This appears to work just fine:

```
div { @include border-radius(1px 5px); }
```

I get the following CSS:

```
div {
  -moz-border-radius: 1px 5px;
  -webkit-border-radius: 1px 5px;
  -o-border-radius: 1px 5px;
  -ms-border-radius: 1px 5px;
  -khtml-border-radius: 1px 5px;
  border-radius: 1px 5px;
}
```

**Comment by chrisdrackett** [comment]
oh, cool. Then I guess the only issue is mozilla with the "global" border radius. if you are specifying it at the top level, they want it like:

```
-moz-border-radius: 1px/5px;
```

although if you are being more specific (calling out, say, top left), it works the same as in webkit:

```
-moz-border-radius-topleft:     4px 2px;
```

**Comment by chriseppstein** [comment]
I've reopened the issue then because we'd need multiple arguments to do different syntaxes like that. Am I correct in assuming that webkit has a space delimited syntax for -webkit-border-radius?

**Comment by chrisdrackett** [comment]
right, so webkit works right now actually, just not mozilla.

**Comment by chriseppstein** [comment]
fixed by c02cab30bb27a7466b389a04320a863824dad690
